lib/img/aws.js

```diff
@@ -45,6 +45,29 @@ AwsDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function awsDelete(key) {
+    var s3 = this._s3;
```

Comment:
In terms of testing, can we mock the s3 object and verify that `deleteObjects` is called on it when `delete` is invoked?